pkg/ark-lib/intent/message.go

```diff
@@ -12,16 +14,14 @@ const (
 	IntentMessageTypeDelete   IntentMessageType = "delete"
 )
 
-type BaseIntentMessage struct {
+var tagHashMessage = []byte("ark-intent-proof-message")
```

Comment:
so we are introducing a prefix to be used when computing the message hash, what's the reason?

Reply:
we're not introducing it, it was already the case for BIP322. I changed from "BIP322-message" (the official one) to "ark-intent-proof-message"